src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1974,27 +1043,28 @@ bool entitiesQuery
         }
       }
     }
-    else if (sco->type == SCOPE_TYPE_SIMPLE_QUERY)
+    else if (scopeP->type == SCOPE_TYPE_SIMPLE_QUERY)
     {
-      // FIXME P4: Once issue #1705 is implemented, this check can be removed.
-      if (qStringFilters(sco->value, filters) != true)
+      for (unsigned int ix = 0; ix < scopeP->stringFilter.mongoFilters.size(); ++ix)
```

Comment:
Where the parse() (that is needed before using the mongoFitlers) is done for this Scope? At parsing stage?